### PR TITLE
Fix build for windows

### PR DESCRIPTION
**Pull request by zoewangg**
Fixing `EnvironmentVariableHelperTest` for windows.

### CODE REVIEW

File: buildspecs/build-windows.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: 0.2
+
+phases:
+  build:
+    commands:
+    - mvn clean install
```
Comment (millems) [inline buildspecs/build-windows.yml]: why not just build.yml?

Comment (zoewangg) [reply buildspecs/build-windows.yml]: `-Dmaven.wagon.httpconnectionManager.maxPerRoute=2` is not working on Windows somehow. Got `Unknown lifecycle phase ".wagon.httpconnectionManager.maxPerRoute=2".` error

Comment (dagnir) [reply buildspecs/build-windows.yml]: I believe it's because there's no space between `-D` and the option. Adding a space should fix it

Comment (zoewangg) [reply buildspecs/build-windows.yml]: Yeah, adding the space fixed it.

**Pull request merged**